src/get-started/beam-overview.md

```diff
@@ -40,6 +40,8 @@ Beam currently supports Runners that work with the following distributed process
          alt="Apache Spark">
 * Google Cloud Dataflow <img src="{{ site.baseurl }}/images/logos/runners/dataflow.png"
          alt="Google Cloud Dataflow">
+* JStorm <img src="{{ site.baseurl }}/images/logos/runners/jstorm.png"
```

Comment:
I think as per @kennknowles only runners in master should be on the overview page, so this should be removed.

Reply:
Done